### PR TITLE
Use GitHub CI

### PR DESCRIPTION
**Pull request by RyanZim**
Travis integration isn't working; this eliminates the need for 3rd-party integrations.

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    strategy:
+      matrix:
+        node-version: ["7.6.0", 8.x, 10.x, 12.x, 14.x, 16.x, 18.x]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
```
Comment (fanatid) [inline .github/workflows/node.js.yml:15]: https://docs.npmjs.com/cli/v8/commands/npm-ci
```suggestion
    - run: npm ci
```

**Review by fanatid — APPROVED** [review]
LGTM, thanks!

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (RyanZim) [reply .github/workflows/node.js.yml:15]: We can't use `npm ci` because we don't commit `package-lock.json`

Comment (fanatid) [reply .github/workflows/node.js.yml:15]: ah, right :+1:

**Pull request merged**